homeassistant/components/ecobee/number.py

```diff
@@ -105,3 +115,46 @@ def set_native_value(self, value: float) -> None:
         """Set new ventilator Min On Time value."""
         self.entity_description.set_fn(self.data, self.thermostat_index, int(value))
         self.update_without_throttle = True
+
+
+class EcobeeAuxCutoverThreshold(EcobeeBaseEntity, NumberEntity):
+    """A number class, representing the minimum outdoor temperature at which the compressor can operate.
+
+    This applies more to air source heat pumps than geothermal. This serves as a safety feature (compressors have a minimum operating temperature) as well as providing the ability to choose fuel in a dual-fuel system (i.e. choose between electrical heat pump and fossil auxiliary heat depending on Time of Use, Solar, etc.).
```

Comment:
Please make sure lines are not longer than 88 characters

Reply:
https://github.com/home-assistant/core/pull/129474/commits/69956f1be5e1c0d161728e26b96d50a7975472e0